### PR TITLE
Add MDs 12466 and 11243 to LHC logbook

### PR DESCRIPTION
**Pull request by fsoubelet**
I've copy-pasted the OP logbook summary of MD12466 into the OMC logbook so we don't have to click around too much.

Tested locally and all links are fine. Would be good if you check my small summary of the HLLHC optics MD looks correct.

### CODE REVIEW

**Comment by fsoubelet** [comment]
Also added post TS1 validation to this PR. All fine locally too.

File: docs/resources/logbook/2024_lhc.md [hunk 1]
```diff
@@ -35,6 +35,8 @@ The tables below can be sorted by clicking next to the column headers.
...
 | 2024-04-09 12:30 | 2024-04-09 14:00 |    0.3W    | Commissioning |                 Measurements with Beam-Beam Wires                  | [Shift Plan](2024-04-09, 1081, 4002607){.logbook-link} / [Summary](2024-04-09, 1081, 4002706){.logbook-link} |
 | 2024-05-14 19:00 | 2024-04-15 04:00 |    1WN     |   MD 11743    | Effect of higher-order contributions to octupole and decapole RDTs | [Shift Plan](2024-05-14, 1081, 4067449){.logbook-link} / [Summary](2024-05-15, 1081, 4067625){.logbook-link} |
+| 2024-06-06 07:00 | 2024-06-06 14:00 | 0.5WN 0.4W |   MD 12466    |                  Effect of CMS Solenoid on Optics                  | [Shift Plan](2024-06-06, 1081, 4082214){.logbook-link} / [Summary](2024-06-06, 1081, 4082930){.logbook-link}  |
+| 2024-06-06 15:00 | 2024-06-07 01:25 | 0.5W 0.3WN |   MD 11243    |                        HL-LHC Optics Part 1                        | [Shift Plan](2024-06-06, 1081, 4082563){.logbook-link} / [Summary](2024-06-07, 1081, 4082920){.logbook-link} |
```
Comment (JoschD) [inline docs/resources/logbook/2024_lhc.md:39]: I don't think I agree with your estimates.
MD 12466 is 
7:00 - 8:30 = 1.5/8 = 0.2 WN
8:30 - 14:00 = 5.5/8 = 0.7 W

MD 11243
15:00 - 17:30 = 2.5/8 =  0.3 W
17:30 - 01:30 = 8/8 = 1 WN

Comment (fsoubelet) [reply docs/resources/logbook/2024_lhc.md]: Thanks, I can't math

File: docs/resources/logbook/2024_lhc.md [hunk 2]
```diff
@@ -55,6 +57,7 @@ The tables below can be sorted by clicking next to the column headers.
...
 *[Measurements with Beam-Beam Wires]: Measured with BB wires at 30cm. Little to no impact seen on coupling and beta-beating. Potential sextupolar RDTs impact. IR5 wire died and we dumped early.
 *[Effect of higher-order contributions to octupole and decapole RDTs]: Test MQ trim. Injection decay measurement. Sensitivity to MCS. Impact of b5 and a4 on the DA and lifetime (LHCB2). MO footprint contraction (LHCB1).
-
+*[Effect of CMS Solenoid on Optics]: Measured with CMS Solenoid at nominal and then ramped down. Measurements at 30cm end-of-MD show a large betabeat degradation compared to final commissioning validation, more than expected. Before / after ramp-down only gives a negigible difference in optics and local coupling. Detuning impact to be checked offline.
+*[HL-LHC Optics Part 1]: Injection optics measured and corrected with success. Ramp was measured and went great except at 1TeV where betabeating goes over 20%. Flat-top optics measured and corrected. Great correction for B2 (incl. dispersion) while B1 needed two knobs (due to exceeding limit of RQTL.9L7.B1), and still shows good control of betas below 10%.
```
Comment (JoschD) [inline docs/resources/logbook/2024_lhc.md:63]: This summary is not really correct. Beam 1 has only a single knob, Beam 2 has two knobs, but this is not well reflected in the logbook (see my email from that night)

**Review by JoschD — APPROVED** [review]

**Pull request merged**